nuget/Microsoft.Windows.CppWinRT.props

```diff
@@ -43,6 +43,20 @@ Copyright (C) Microsoft Corporation. All rights reserved.
             <ProxyFileName Condition="'%(Midl.ProxyFileName)'==''">nul</ProxyFileName>
             <TypeLibraryName Condition="'%(Midl.TypeLibraryName)'==''"></TypeLibraryName>
         </Midl>
+        <ProjectReference Condition="'$(XamlLanguage)' != 'C++'">
```

Comment:
these are defaults.  if necessary, users could explicitly override in the IDE?

Reply:
Correct. Overrides are possible per reference, for all references or conditionally.